### ISSUE TITLE
[Bug]: LTX-Video bf16 on CPU produces grey/bad quality videos

### ISSUE THREAD

**Issue by goyaladitya05** [openvinotoolkit/openvino#36715]
### OpenVINO Version

Master

### Operating System

Ubuntu 20.04 (LTS)

### Device used for inference

CPU

### Framework

None

### Model used

https://huggingface.co/Lightricks/LTX-Video

### Issue description

On a bf16 capable CPU, Text2VideoPipeline (LTX-Video) produces grey/bad quality videos, where default precision is bf16. With fp32 and the same seed, the output is good. 

1. The outputs are bad on both sides (OpenVINO GenAI and Optimum), which indicates this must be an issue with how the OpenVINO CPU Plugin handles bf16.
2. GPU on fp16 works fine. CPU on fp32 works fine.
3. Pytorch bf16 works fine. 

### Step-by-step reproduction

```
#!/usr/bin/env python3
"""LTX-Video Text-to-Video Sample"""

import torch
from optimum.intel import OVPipelineForText2Video
from diffusers.utils import export_to_video

pipe = OVPipelineForText2Video.from_pretrained("Lightricks/LTX-Video", export=True, device="CPU")

prompt = (
    "A golden retriever in a sunlit bedroom slowly stands up, stretches its front paws forward, "
    "yawns wide showing pink tongue, then turns its head to look at the camera, ears perking up alertly, "
    "tail beginning to wag, warm afternoon window light, cinematic shallow depth of field, "
    "smooth handheld camera motion, photorealistic, detailed fur"
)
negative_prompt = (
    "static, motionless, frozen, still photograph, no movement, low quality, blurry, distorted, "
    "deformed limbs, extra paws, morphing, melting, choppy motion"
)

video = pipe(
    prompt=prompt,
    negative_prompt=negative_prompt,
    num_frames=97,
    height=480,
    width=704,
    num_inference_steps=50,
    guidance_scale=4.0,
    generator=torch.Generator().manual_seed(42),
).frames[0]

export_to_video(video, "ltx_t2v_output.mp4", fps=25)
print(f"video saved! Frames: {len(video)}, Duration: {len(video)/25:.1f}s")

```

### Issue submission checklist

- [x] I'm reporting an issue. It's not a question.
- [x] I checked the problem with the documentation, FAQ, open issues, Stack Overflow, etc., and have not found a solution.
- [x] There is reproducer code and related data files such as images, videos, models, etc.

**Comment by goyaladitya05** [comment]
Output generated on CPU with bf16 ->
https://github.com/user-attachments/assets/b677d651-9652-4158-8914-43ca638794cf

Output generated on CPU with fp32 ->
https://github.com/user-attachments/assets/03a6b4e6-6acb-4f1d-8049-8ad0fb954f07

**Comment by goyaladitya05** [comment]
#36718 and #36719 combined are able to fix this issue. I validated by building OpenVINO from these pull requests and running the reproduction script, obtained output is attached. Quality improved a lot, inference time was increased by about ~1 sec, but the speedup against fp32 is still almost 3.8x.

https://github.com/user-attachments/assets/1534f509-b157-439b-b669-7e53d92b6b0f

**Comment by goyaladitya05** [comment]
Closing since both #36718 and #36719 has been merged.
